README.md

```diff
@@ -19,6 +19,9 @@ DELETE /v1/ecs/{account}/clusters/{cluster}/services/{service}[?recursive=true]
 GET /v1/ecs/{account}/clusters/{cluster}/services/{service}
 GET /v1/ecs/{account}/clusters/{cluster}/services/{service}/events
 GET /v1/ecs/{account}/clusters/{cluster}/services/{service}/logs?task="foo"&container="bar"
+GET /v1/ecs/{account}/clusters/{cluster}/services/{service}/logs?task="foo"&container="bar"&limit="30"&seq="f/35313851203912372440587619261645128276299525300062978048"
+GET /v1/ecs/{account}/clusters/{cluster}/services/{service}/logs?task="foo"&container="bar"&start="1583504305223"&end="1583527860973"
+GET /v1/ecs/{account}/clusters/{cluster}/services/{service}/logs?task="foo"&container="bar"&start="1583504305223"&end="1583527860973"&limit="30"&seq="f/35313851203912372440587619261645128276299525300062978048"
```

Comment:
Maybe just have the actual endpoint here: `GET /v1/ecs/{account}/clusters/{cluster}/services/{service}/logs`
And then a separate section with all the different params and how to use them. Not being familiar with the syntax, it's unclear how to specify the start/end, and what's a limit and seq. A link to the CloudWatch docs might be sufficient.

Reply:
I added them explicitly because those parameters are required to be specified that way or the routes won't match.  I'll add a link to the doc too.  It looks scarier than it is -- the times are milliseconds since the epoch and the seq is just a token given to you in the request.

Reply:
ok i worked this out - im parsing the queries myself instead of using gorilla to do it... its more flexible